src/app/components/observation/observation-image-carousel/observation-image-carousel.component.html

```diff
@@ -0,0 +1,80 @@
+<ng-container>
```

Comment:
Hvorfor trengs ng-container her, antar den kan fjernes?

Reply:
den trengs fordi jeg har flere elementer <swiper-container>, <div class="image-description"> og  <button> som  high-level elementer, og for å unngå en ekstra div wrapper, brukte jeg kun ng-container.

Reply:
Hva trenger du wrapperen til?

Reply:
tydeligvis det er det :P . jeg får feil hvis jeg har to hoved elementer.

Reply:
hmm har testet igjen i dag, og i dag fikk jeg ikke et feil... super rart xd

Reply:
Skal gå fint å ha flere hovedelementer med angular, så vidt jeg vet. Feks observation.component har det, sikkert mange andre og.